sn_networking/src/record_store.rs

```diff
@@ -1729,7 +1729,7 @@ mod tests {
 
             // Check termination condition
             if hour == max_hours {
-                let acceptable_percentage = 0.01; //%
+                let acceptable_percentage = 0.12; //%
```

Comment:
so with our change we end up with loads more empty nodes? 🤔 

hmm. This number feels a bit magic and a bit high.